content/en/docs/operations/virtualization/gpu.md

```diff
@@ -0,0 +1,191 @@
+    ```
+
+You can use `kubectl debug node` or `kubectl node-shell -x` to access the node and check the GPU status.
+
+```bash
+lspci --nnk -d 10de:
+```
+
+The vfio-manager pod will bind all GPUs on the node to the vfio-pci driver. Example output:
+
+```console
+3b:00.0 3D controller [0302]: NVIDIA Corporation Device [10de:2236] (rev a1)
+       Subsystem: NVIDIA Corporation Device [10de:1482]
+       Kernel driver in use: vfio-pci
+86:00.0 3D controller [0302]: NVIDIA Corporation Device [10de:2236] (rev a1)
+       Subsystem: NVIDIA Corporation Device [10de:1482]
+       Kernel driver in use: vfio-pci
+```
+
+The sandbox-device-plugin will discover and advertise these resources to kubelet.
+In this example, we have two A10 GPUs:
+
+```bash
+kubectl describe node <node-name>
+```
+
+Example output:
+
+```console
+...
+Capacity:
+  ...
+  nvidia.com/GA102GL_A10:         2
+  ...
+Allocatable:
+  ...
+  nvidia.com/GA102GL_A10:         2
+...
+```
+
+{{% alert color="info" %}}
+The resource name is currently constructed by joining the device and device_name columns from the [PCI IDs database](https://pci-ids.ucw.cz/v2.2/pci.ids).
+For example, the entry for A10 in the database reads `2236  GA102GL [A10]`, which results in a resource name `nvidia.com/GA102GL_A10`.
+{{% /alert %}}
+
+## 2. Update the KubeVirt Custom Resource
+
+Next, we will update the KubeVirt Custom Resource, as documented in the 
+[KubeVirt user guide](https://kubevirt.io/user-guide/virtual_machines/host-devices/#listing-permitted-devices),
+so that the passthrough GPUs are permitted and can be requested by a KubeVirt VM.
+
+Replace the values for `pciVendorSelector` and `resourceName` to correspond to your GPU model.
+Setting `externalResourceProvider=true` indicates that this resource is being provided by an external device plugin,
+in this case the `sandbox-device-plugin` which is deployed by the Operator.
+
+```bash
+kubectl edit kubevirt -n kubevirt
+```
+example config:
+```yaml
+  ...
+  spec:
+    permittedHostDevices:
+      pciHostDevices:
+      - externalResourceProvider: true
+        pciVendorSelector: 10DE:2236
+        resourceName: nvidia.com/GA102GL_A10
+  ...
+```
+
+## 3. Create a Virtual Machine
+
+We are now ready to create a VM. Let’s create a sample VM using a simple VMI spec which requests a `nvidia.com/GA102GL_A10` resource.
+
+**vmi-gpu.yaml**:
+
+```yaml
+---
+apiVersion: apps.cozystack.io/v1alpha1
+appVersion: '*'
+kind: VirtualMachine
+metadata:
+  name: gpu
+  namespace: tenant-example
+spec:
+  running: true
+  instanceProfile: ubuntu
+  instanceType: u1.medium
+  systemDisk:
+    image: ubuntu
+    storage: 5Gi
+    storageClass: replicated
+  gpus:
+  - name: nvidia.com/GA102GL_A10
+  cloudInit: |
+    #cloud-config
+    password: ubuntu
+    chpasswd: { expire: False }
+```
+
+```bash
+kubectl apply -f vmi-gpu.yaml
+```
+
+example output:
+```console
+virtualmachines.apps.cozystack.io/gpu created
+```
+
+```bash
+k get vmi
```

Comment:
@kvaps what is `k`, some alias to `kubectl`?

Reply:
yes, thank you for noticing this